java/server/src/org/openqa/selenium/grid/graphql/Node.java

```diff
@@ -33,18 +38,29 @@
   private final int maxSession;
   private final String capabilities;
   private static final Json JSON = new Json();
+  private final Set<Session> activeSessions;
 
 
   public Node(UUID id,
               URI uri,
               boolean isUp,
               int maxSession,
-              Map<Capabilities, Integer> capabilities) {
+              Map<Capabilities, Integer> capabilities,
+              Set<Session> activeSessions) {
```

Comment:
The list of active sessions is internal state to the node.

Reply:
Closing, had a chat on Slack regarding this.

Reply:
What was the conclusion of that Slack conversation?

Reply:
There was some confusion regarding which node file it was.